src/main/preload/mattermost.js

```diff
@@ -140,7 +140,7 @@ window.addEventListener('message', ({origin, data = {}} = {}) => {
     case 'dispatch-notification': {
         if (shouldSendNotifications) {
             const {title, body, channel, teamId, url, silent, data: messageData} = message;
-            ipcRenderer.send(NOTIFY_MENTION, title, body, channel, teamId, url, silent, messageData);
+            ipcRenderer.send(NOTIFY_MENTION, title, body, channel, teamId, url, silent, messageData?.soundName);
```

Comment:
Is there anything else that we could end up putting in `messageData` that might be useful here?

I actually forget what's in there besides the sound name :P